prusti-tests/tests/parse/ui/no-attr-params.rs

```diff
@@ -13,7 +13,9 @@ fn pure_err() -> bool {
 #[trusted(dont write things here)]
 fn trusted_err() {}
 
-#[predicate(dont put anything here)]
-fn pred_err() {}
+predicate! {
+    #[dont_put_anything_here]
+    fn pred_err() {}
+}
```

Comment:
I think this test can just be deleted now.

Reply:
Sorry, I should have been clearer – I meant the highlighted lines, the remainder of the file is still good.

Reply:
ah, fixed.